main/character.py

```diff
@@ -18,17 +19,39 @@ def __init__(self, x, y, name, strength, defense, health_points):
         # self.rect = self.image.get_rect()
         # self.rect.center = (x, y)
 
-    def attack(self, target: 'Character'):
```

Comment:
I think attack method shouldn't be deleted and should be used inside inside light-medium-strong_attack. Attack should have addional argument (type_attack_multiplier) for multiplication `base_damage()`.Whole light-medium-strong_attack methods should looks like that: 
`def light_attack(self, target: "Character",type_attack_multiplier ):` 
      `self.attack(type_attack_multiplier)` 
Maybe it a good idea to make reference type_attack_multiplier in config.ini